src/binary-writer.cc

```diff
@@ -1002,8 +1009,12 @@ Result BinaryWriter::WriteModule(const Module* module) {
   }
 
   if (module->excepts.size()) {
-    // TODO(karlschimpf) Define.
-    WABT_FATAL("write exception section not implemented");
+    BeginCustomSection("exception", LEB_SECTION_SIZE_GUESS);
+    write_u32_leb128(&stream_, module->excepts.size(), "exception count");
```

Comment:
The spec at the moment does not introduce a new builtin section, and there were people who fought back against making it a builtin section.

However, since we are still a long way away from getting the proposal approved, it may change to a builtin section in the future.